content/english/security/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Spy Game"
+description: "An introductory guide to learn about passwords and how to create a strong one!"
+date: 2020-02-217T11:45:38-07:00
+prereq: "None"
+difficulty: "Beginner"
+download: ""
+draft: true
+icon: "fas fa-microchip"
+weight: 1
+translationKey: "security-main"
+---
+
+1. [Meeting Alice, Bob and Eve](1-meeting-alice-bob-and-eve)
```

Comment:
consider adding {{% children %}} instead of listing it. Refer other workshops like https://github.com/NuevoFoundation/workshops/blob/master/content/english/python-earsketch/_index.md

Reply:
made the change!